### ISSUE TITLE
gherkin-official==4.1.0 can't find gherkin-languages.json file 

### ISSUE THREAD

**Issue by electroniceagle** [koterpillar/aloe#127]
This update to gherkin-official just dropped and it causes aloe to fail on import.

```
$ make test SITE=awesim
make -C awesim test
./manage.py test --keepdb
Traceback (most recent call last):
  File "./manage.py", line 18, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/django/core/management/__init__.py", line 302, in execute
    settings.INSTALLED_APPS
  File "/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/django/conf/__init__.py", line 55, in __getattr__
    self._setup(name)
  File "/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/django/conf/__init__.py", line 43, in _setup
    self._wrapped = Settings(settings_module)
  File "/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/django/conf/__init__.py", line 99, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/Users/bschott/Source/sites/awesim/site_awesim/__init__.py", line 7, in <module>
    from .celery import app as celery_app  # noqa
  File "/Users/bschott/Source/sites/awesim/site_awesim/celery.py", line 15, in <module>
    django.setup()
  File "/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/django/apps/config.py", line 90, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/aloe_django/__init__.py", line 19, in <module>
    from aloe.testclass import TestCase as AloeTestCase
  File "/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/aloe/testclass.py", line 24, in <module>
    from aloe.parser import (
  File "/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/aloe/parser.py", line 19, in <module>
    from gherkin.dialect import Dialect
  File "/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/gherkin/dialect.py", line 10, in <module>
    with io.open(DIALECT_FILE_PATH, 'r', encoding='utf-8') as file:
IOError: [Errno 2] No such file or directory: '/Users/bschott/.virtualenvs/sites/lib/python2.7/site-packages/gherkin/gherkin-languages.json'
make[1]: *** [test] Error 1
make: *** [test] Error 2
```

**Comment by koterpillar** [comment]
cucumber/cucumber#153. [Pin your packages](https://github.com/jazzband/pip-tools) in meanwhile!

**Comment by koterpillar** [comment]
Fixed in `gherkin-official==4.1.1`.

**Comment by electroniceagle** [comment]
I just confirmed gherkin-official==4.1.1. is on pypi and appears to resolve the issue.
